iidm/iidm-modification/src/main/java/com/powsybl/iidm/modification/AbstractNetworkModification.java

```diff
@@ -91,4 +95,33 @@ protected void logOrThrow(boolean throwException, String message) {
             LOGGER.warn("Error while applying modification : {}", message);
         }
     }
+
+    @Override
+    public NetworkModificationImpact hasImpactOnNetwork(Network network) {
+        return DEFAULT_IMPACT;
+    }
+
+    protected static boolean checkVoltageLevel(Identifiable<?> identifiable) {
+        VoltageLevel vl;
+        if (identifiable instanceof Bus bus) {
+            vl = bus.getVoltageLevel();
+        } else if (identifiable instanceof BusbarSection bbs) {
+            vl = bbs.getTerminal().getVoltageLevel();
+        } else {
+            return false;
+        }
+        return vl != null;
+    }
+
+    protected boolean areValuesEqual(Double newValue, double currentValue, boolean isRelativeValue) {
+        return newValue == null || Math.abs(newValue - (isRelativeValue ? 0 : currentValue)) < EPSILON;
+    }
+
+    protected boolean areValuesEqual(Integer newValue, int currentValue, boolean isRelativeValue) {
+        return newValue == null || Math.abs(newValue - (isRelativeValue ? 0 : currentValue)) < EPSILON;
+    }
+
+    protected boolean isValueOutsideRange(Integer newValue, int minValue, int maxValue) {
+        return newValue == null || newValue < minValue || newValue > maxValue;
```

Comment:
newValue == null : here  => isValueOutsideRange true so probably can be applied false while if no value can be applied should be true. (or we can remove the null case and used int)